explorer/client/src/serviceWorker.ts

```diff
@@ -22,7 +22,7 @@ const isLocalhost = Boolean(
     ),
 )
 
-export function register(config) {
+export function register(config: any) {
```

Comment:
How do you feel about leaving this file as JS? It's generated by CRA so if they update it on their side, we can just use theirs